File: VB/MyTabControl/MyTabControl.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraTab
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace Example_HidePageHeader.MyTabControl
	Public Class MyTabControl
		Inherits XtraTabControl
		Protected Overrides Function CreateTabCollection() As XtraTabPageCollection
			Return New MyXtraTabPageCollection(Me)
		End Function
	End Class
End Namespace

File: CS/MyTabControl/MyTabControl.cs
using DevExpress.XtraTab;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Example_HidePageHeader.MyTabControl {
    public class MyTabControl : XtraTabControl {
        protected override XtraTabPageCollection CreateTabCollection() {
            return new MyXtraTabPageCollection(this);
        }
    }
}
